src/nightwatch/responsive-helpers.js

```diff
@@ -0,0 +1,39 @@
+const screenWidth = (browser) => {
+  const breakpoint = browser.currentTest.name.match(/\[@(.*?)\]/)[1];
```

Comment:
This is a good idea.. can I suggest we turn this into a [custom command](http://nightwatchjs.org/guide#writing-custom-commands)?   

```js
const breakpoint = browser.breakpoint();
```

It may also be nice to have custom commands that do something similar to

```js
browser.whenScreenSize('small', () => {
  // todo assert things when screensize is small
});
```

Don't have to name the function exactly that, but a utility to make it easier to scope tests. Once could also argue that rather than that function.. the assertions should just be broken into separate tests that only resize to what they are asserting.

Reply:
I'm ok with leaving this as a potential enhancement in the future for now.